spec/features/assignments/index_spec.rb

```diff
@@ -10,54 +10,65 @@
     set_current_user(user)
   end
 
+  after :each do
+    Timecop.return
+  end
+
   it 'highlights today' do
     visit roster_assignments_url(roster)
     expect(page).to have_selector('td.cal-cell.current-day', text: '14')
   end
   context 'assignment belongs to user' do
-    it 'appears highlighted yellow for your assignment' do
-      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
+    it 'appears highlighted for your assignment' do
+      create :assignment, start_date: 3.days.ago, end_date: 3.days.since,
                           user: user, roster: roster
       visit roster_assignments_url(roster)
-      expect(page).to have_selector('td .cal-event.assignment-user', count: 5)
+      expect(page).to have_selector('td .cal-event.assignment-user', count: 7)
     end
   end
 
   context 'assignment does not belong to user' do
-    it 'appears highlighted blue for other assignments' do
-      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
+    it 'appears highlighted differently for other assignments' do
+      create :assignment, start_date: 3.days.ago, end_date: 3.days.since,
                           user: roster_user(roster), roster: roster
       visit roster_assignments_url(roster)
-      expect(page).to have_selector('td .cal-event.assignment', count: 5)
+      expect(page).to have_selector('td .cal-event.assignment', count: 7)
     end
   end
 
   context 'start of assignment' do
-    it 'has a left radius and a smaller width' do
-      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
+    it 'has a left radius' do
+      create :assignment, start_date: 3.days.ago, end_date: 3.days.since,
                           user: user, roster: roster
       visit roster_assignments_url(roster)
       expect(page).to have_selector('td .cal-event.assignment-start', count: 1)
     end
   end
 
   context 'end of assignment' do
-    it 'has a right radius' do
-      create :assignment, start_date: 2.days.ago, end_date: 2.days.since,
+    it 'has a right radius and a smaller width than the cell' do
```

Comment:
I can't tell that the end of the assignment has a smaller width from the test. You've moved this from line 36 to line 49. Just wanted to make sure you did that on purpose.